src/gui/mainwindow.cpp

```diff
@@ -119,7 +119,7 @@ namespace
     }
 }
 
-MainWindow::MainWindow(IGUIApplication *app, QWidget *parent)
+MainWindow::MainWindow(IGUIApplication *app, QWidget *parent, bool startMinimizedToTray)
```

Comment:
1. I think we can remove `parent` parameter from `MainWindow` constructor
2. You really override `startMinimized` setting so `ToTray` is redundant/confusing suffix
3. Boolean parameters for such use are discouraged, using enum makes caller side more readable, e.g. `new MainWindow(this, Minimized);`

Reply:
The comment above is still relevant.

Reply:
1. Will removing QWidget *parent from mainwindow.app line 122 and mainwindows.h line 82 do?
2. So should I just name it startMinimized? I just don't want to confuse the users that'll use the argument.
3. Could you give an example of the declaration of this enum?